net/snode.c/Makefile

```diff
@@ -0,0 +1,275 @@
+               -DTERMINATE_TIMEOUT=$(CONFIG_SNODEC_TERMINATE_TIMEOUT) \
+               -DBACKLOG=$(CONFIG_SNODEC_BACKLOG) \
+               -DACCEPTS_PER_TICK=$(CONFIG_SNODEC_ACCEPTS_PER_TICK) \
+               -DREUSE_ADDRESS=$(CONFIG_SNODEC_REUSE_ADDRESS) \
+               -DREUSE_PORT=$(CONFIG_SNODEC_REUSE_PORT) \
+               -DIPV6_ONLY=$(CONFIG_SNODEC_IPV6_ONLY) \
+               -DIPV4_MAPPED=$(CONFIG_SNODEC_IPV4_MAPPED) \
+               -DACCEPT_TIMEOUT=$(CONFIG_SNODEC_ACCEPT_TIMEOUT) \
+               -DCONNECT_TIMEOUT=$(CONFIG_SNODEC_CONNECT_TIMEOUT) \
+               -DRECONNECT=$(CONFIG_SNODEC_RECONNECT) \
+               -DRECONNECT_TIME=$(CONFIG_SNODEC_RECONNECT_TIME) \
+               -DRETRY=$(CONFIG_SNODEC_RETRY) \
+               -DRETRY_TIMEOUT=$(CONFIG_SNODEC_RETRY_TIMEOUT) \
+               -DRETRY_TRIES=$(CONFIG_SNODEC_RETRY_TRIES) \
+               -DRETRY_BASE=$(CONFIG_SNODEC_RETRY_BASE) \
+               -DRETRY_JITTER=$(CONFIG_SNODEC_RETRY_JITTER) \
+               -DRETRY_LIMIT=$(CONFIG_SNODEC_RETRY_LIMIT) \
+               -DTLS_INIT_TIMEOUT=$(CONFIG_SNODEC_TLS_INIT_TIMEOUT) \
+               -DTLS_SHUTDOWN_TIMEOUT=$(CONFIG_SNODEC_TLS_SHUTDOWN_TIMEOUT) \
+               -DCMAKE_BUILD_WITH_INSTALL_RPATH=ON \
+               -DCMAKE_SKIP_RPATH=FALSE
+
+define Package/snode.c/Default
+  SECTION:=net
+  CATEGORY:=Network
+  SUBMENU:=SNode.C
+  URL:=https://github.com/VolkerChristian/snode.c
+  Maintainer:=Volker Christian <me@vchrist.at>
+endef
+
+define Package/snode.c
+  $(call Package/snode.c/Default)
+  TITLE:=Easy to use lightwight layer based network framework for C++
+  DEPENDS:=+libstdcpp
+endef
+
+define Package/snode.c/description
+  SNode.C is a very simple to use lightweight highly extendable event
+driven layer-based framework for network applications in the spirit
+of node.js written entirely in C++.
+
+endef
+
+define Package/snode.c/config
+  source "$(SOURCE)/Config.in"
+endef
+
+define Package/snode.c/install
+	true
+endef
+
+SNODEC_GROUP_NAME=snodec
+SNODEC_GROUP_ID=$(call qstrip,$(CONFIG_SNODEC_GROUP_ID))
+
+define Package/snode.c/postinst
+#!/bin/sh
+
+GROUP_NAME=$(SNODEC_GROUP_NAME)
+GROUP_ID=$(SNODEC_GROUP_ID)
+
+if [ -z $$(grep "^$${GROUP_NAME}:" $${IPKG_INSTROOT}/etc/group) ]; then
+	echo "Adding group '$${GROUP_NAME}' with ID $${GROUP_ID} to /etc/group and assign user root to it"
+	echo "$${GROUP_NAME}:x:$${GROUP_ID}:root" >> $${IPKG_INSTROOT}/etc/group
+else
+	echo "Adding group '$${GROUP_NAME}' to /etc/group failed: group exists"
+fi
+endef
+
+define Package/snode.c/prerm
+#!/bin/sh
+
+GROUP_NAME=$(SNODEC_GROUP_NAME)
+GROUP_ID=$(SNODEC_GROUP_ID)
+
+if [ ! -z $$(grep "^$${GROUP_NAME}:.*:$${GROUP_ID}" $${IPKG_INSTROOT}/etc/group) ]; then
+	echo "Removing group '$${GROUP_NAME}' from /etc/group"
+	sed -i "/^$${GROUP_NAME}:/d" /etc/group
+else
+	echo "Removing group '$${GROUP_NAME}' from /etc/group failed: no such group"
+fi
+endef
```

Comment:
Same as in Config.in. User and group can be created differently and much simpler.

Reply:
Already mentioned above - done.